src/main/java/invtweaks/container/DirectContainerManager.java

```diff
@@ -82,8 +82,8 @@ public boolean move(ContainerSection srcSection, int srcIndex, ContainerSection
 
         // Use intermediate slot if we have to swap tools, maps, etc.
         assert srcStack != null;
-        if(destStack != null && srcStack.getItem() == destStack.getItem() && (srcStack.getMaxStackSize() == 1 ||
-                srcStack.hasTagCompound() || destStack.hasTagCompound())) {
+        if(destStack != null && srcStack.isItemEqual(destStack) && srcStack.getMaxStackSize() == 1 &&
+                ((srcStack.hasTagCompound() || destStack.hasTagCompound()) && (srcStack.getTagCompound().equals(destStack.getTagCompound())))) {
```

Comment:
OK, I will still using this patch until you fix that problem. because I has a server that contain lots of stackable item with tag. without this problem. that will be a huge problem in sort.